baselines/jft/input_utils.py

```diff
@@ -25,30 +25,25 @@
 import numpy as np
 import tensorflow as tf
 import tensorflow_datasets as tfds
-import al_utils  # local file import from baselines.jft
 
-SubsetDatasetBuilder = al_utils.SubsetDatasetBuilder
+T_DatasetBuilder = TypeVar("T_DatasetBuilder", bound=deterministic_data.DatasetBuilder)
```

Comment:
By the way, lint checks failed for `T_DatasetBuilder`, so I updated to `DatasetBuilderType`.